pkg/util/cluster_test.go

```diff
@@ -95,7 +104,7 @@ var _ = Describe("test clusterInfo", func() {
 		os.Setenv(OperatorConditionNameEnvVar, "aValue")
 		cl := fake.NewClientBuilder().
 			WithScheme(testScheme).
-			WithRuntimeObjects(clusterVersion, infrastructure, ingress, apiServer).
+			WithRuntimeObjects(clusterVersion, infrastructure, ingress, apiServer, dns).
```

Comment:
Maybe we can add a test for `GetBaseDomain` and `getClusterBaseDomain` here, or below?

Reply:
I saw it is missing indeed, but saw that there are no UT for the `GetDomain`, `getClusterDomain`
so first wanted to see first comments
Thanks